Sources/Features/DappsAndPersonas/PersonaDetails/PersonaDetails.swift

```diff
@@ -175,6 +178,7 @@ public struct PersonaDetails: Sendable, FeatureReducer {
 				loggerGlobal.error("Could not get dApp details \(dAppID), error: \(error)")
 				errorQueue.schedule(error)
 			}
+			return .none
```

Comment:
hmm this `return .none` will never be executed right?

Reply:
same for this, merge leftover